src/core/tooltip_text.py

```diff
@@ -0,0 +1,14 @@
+RANDOM = 'Check this box to randomly generate an safety car event.'
+MAXIMUM_OCC = 'The maximum number of times a safety car event could potentially occur (not guaranteed).'
+PROBABILITY = 'The likelihood of a safety car being deployed. A value of 1 indicates a 100% chance of a safety car being deployed, while a value of 0 means there is no probability of a safety car.'
+CARS_STOPPED_ON_TRACK = 'Check this box to trigger a safety car event when cars are stopped on the track.'
+MINIMUM_STOPPED_TO_TRIGGER = 'Minimum number of stopped cars on the track required to deploy a safety car.'
+DRIVER_DISPLAY_MESSAGE = 'Message to be displayed to the drivers.'
+CARS_OFF_TRACK = 'Check this box to trigger a safety car event when cars go off track.'
+MINIMUM_OFF_TRACK_TO_TRIGGER = 'Minimum number of off-track cars required to deploy a safety car.'
+MAXIMUM_SAFTEY_CARS = 'Maximum number of safety car deployments allowed during the race.'
+EARLIST_POSSIBLE = 'Earliest time a safety car can be deployed after the race starts (in minutes).'
+LATEST_POSSIBLE = 'Latest time a safety car can be deployed after the race starts (in minutes).'
+MINIMUM_INTERVAL = 'Minimum time interval before another safety car can be deployed.'
+LAPS_UNDER = 'Maximum number of laps a safety car remains deployed.'
+IMMEDIATE_WAVE_AROUND = 'Immediate wave around'
```

Comment:
The immediate wave around feature sends a wave around command to all lapped cars the moment a safety car is thrown. In the future, I'm going to add a delayed wave around feature, but for now, this one is the only option.